main.py

```diff
@@ -1,6 +1,7 @@
 # NOTE: to easily play with the data, use the below and access client_instance
 # $ python3 -i main.py
-from src import client_instance
+from src import _Client
 
 if __name__ == "__main__":
-    client_instance.main()
+	client = _Client()
+	client.main()
```

Comment:
Could you name the instance `client_instance`? That's what we have it as in the externally facing documentation.

Reply:
Sure. Regarding above comment, that's a valid point. If I have time before Tuesday, I'll make a change (shouldm't be too long)